pgctl/cli.py

```diff
@@ -347,7 +346,7 @@ def services(self):
             for alias in self.pgconf['services']
             for service_name in self._expand_aliases(alias)
         ]
-        return uniq(services)
+        return tuple(sorted(set(services)))
```

Comment:
I'm surprised that 

  a) services are even sortable
  b) this doesn't change the output of any test